desktop/client.js

```diff
@@ -15,7 +15,13 @@ http.get('http://localhost:4000/dist/main.bundle.js', function (response) {
   response.pipe(file)
   file.on('finish', function() {
     file.close(function () {
-      const e = spawn(electron, [name], {stdio: 'inherit'})
+      const params = [name]
+
+      if (process.env.USE_INSPECTOR) {
+        params.unshift('--debug-brk=5858')
```

Comment:
That's what I'm talking about. Look at the docs for node inspector. It
starts In a debugged state so you can put in break points to debug startup
stuff

https://www.npmjs.com/package/node-inspector


On Friday, February 5, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In desktop/client.js
> <https://github.com/keybase/client/pull/1915#discussion_r52091958>:
>
> > @@ -15,7 +15,13 @@ http.get('http://localhost:4000/dist/main.bundle.js', function (response) {
> >    response.pipe(file)
> >    file.on('finish', function() {
> >      file.close(function () {
> > -      const e = spawn(electron, [name], {stdio: 'inherit'})
> > +      const params = [name]
> > +
> > +      if (process.env.USE_INSPECTOR) {
> > +        params.unshift('--debug-brk=5858')
>
> I mean the --debug-brk
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1915/files#r52091958>.
>